dotcom-rendering/playwright.config.ts

```diff
@@ -0,0 +1,82 @@
+import { defineConfig, devices } from '@playwright/test';
+
+/**
+ * Read environment variables from file.
+ * https://github.com/motdotla/dotenv
+ */
+// require('dotenv').config();
+
+/**
+ * See https://playwright.dev/docs/test-configuration.
+ */
+// eslint-disable-next-line import/no-default-export -- default export required
```

Comment:
Should this be added as in the ESLint config file for any file matchin `*.config.*`?